portfolio/src/main/webapp/scripts/comments.js

```diff
@@ -2,7 +2,10 @@
  * Fetches data from java servlet and adds it to DOM tree
  */
 async function fetchAndInsertDataIntoDOM() {
-	const response = await fetch('/data');
+	const queryString = constructQueryString();
+	console.log('queryString:', queryString);
```

Comment:
Is this for debugging? Perhaps you don't need it?

Reply:
You're right, it was for debugging. I removed it now.